public/sass/custom/kth-klaro.scss

```diff
@@ -203,43 +123,76 @@
         }
       }
 
+      a {
+        color: #036eb8;
+        text-decoration: underline;
+        text-underline-offset: 0.25rem;
+      }
+
+      //Header section
       .cm-header {
         padding: 1.5rem;
 
-        h1 {
-          &.title {
-            font-size: 1.875rem;
-            font-weight: 400;
-            color: $black;
-            line-height: 1.3125;
-            margin-top: 0;
-            margin-bottom: 0.875rem;
+        .hide {
+          svg {
+            stroke: $black;
           }
         }
+
+        .title {
+          font-size: 1.875rem;
+          font-weight: 400;
+          color: $black;
+          line-height: 1.3125;
```

Comment:
Should the unit be rem? It feels like a quite strange line-height (since it is lower than the font-size)

Reply:
Hm, I'll try and remove it and see if Robert have feedback later.

Reply:
I don't mean that you should remove it, I mean that we should use a better value. 1.3125 means that we multiply that with the font size (which equals 2,4609...rem). For font-size 1.875rem in style, we use 2.5rem in line-height